src/test/java/org/example/HomePageTests.java

```diff
@@ -21,30 +22,46 @@
 @DisplayName("Example of a Test Class from Home Page Module")
 public class HomePageTests extends BaseTests {
 	private static final Logger logger = (Logger) LogManager.getLogger(BaseTests.class);
+	private static final SlackService slackNotifier = new SlackService();
 
 	@Test
 	@DisplayName("Validation of Page Title")
 	public void validatePageTitle() {
 		logger.info("Assert Page Title");
+		slackNotifier.postNotification("Test started: Validation of Page Title");
 		String expectedTitle = "Welcome to the-internet";
 		assertEquals(homePage.getDashboardTitle(), expectedTitle, "Page title does not match.");
 		logger.info("Page title validated");
+		try {
+            slackNotifier.postNotification("Page title validated");
```

Comment:
This approach will provide the notification through SLack about each test result, IMO it is better to set the notification(s) into the @AfterTest and sent let's say the link to the result or whatever will help to understood how the tests are goes. What do you think, is it make sense?

Reply:
Please reconsider it for all the tests. Why I think so, because it will be annoying to receive the notification for each test, if there is 1000 of tests and they will run in parallel, is this notification will give you any useful info?

Reply:
I got the point, thanks)
slackTestNotifier Method implemented in BaseTests Class!